scripts/nayduck.py

```diff
@@ -182,11 +182,16 @@ def run_locally(tests):
         # See nayduck specs at https://github.com/near/nayduck/blob/master/lib/testspec.py
         fields = test.split()
 
+        timeout = ""
         index = 1
+        ignored = []
         while len(fields) > index and fields[index].startswith('--'):
+            if fields[index].startswith("--timeout="):
+                timeout = fields[index].split("=")[1]
+            else:
```

Comment:
Might just as well do `elif fields[index] != '--skip-build':` so as to not bother people about that flag. ‘Skip build’ is the only behaviour this script handles after all.